internal/driver/webhtml.go

```diff
@@ -91,34 +72,56 @@ button {
   box-shadow: 2px 2px 2px 0px #aaa;
   z-index: 1;
 }
+#actionbox {
+  display: none;
+  position: fixed;
+  background-color: #ffffff;
+  border: 1px solid black;
+  box-shadow: 2px 2px 2px 0px #aaa;
+  padding: 1em;
+  top: 20px;
+  right: 20px;
+}
+#actionbox > button {
+  display: block;
+  width: 100%;
+}
+#home {
+  font-size: 24px;
+  padding-left: 0.5em;
+  padding-right: 0.5em;
+}
 </style>
 </head>
 <body>
-<h1>{{.Title}}</h1>
-<div id="page">
-
-<div id="errors">{{range .Errors}}<div>{{.}}</div>{{end}}</div>
 
-<div id="header">
-<div id="leftbuttons">
 <button id="details">&#x25b7; Details</button>
 <div id="detailtext">
 {{range .Legend}}<div>{{.}}</div>{{end}}
 </div>
-<button id="list">List</button>
-<button id="disasm">Disasm</button>
-<input id="searchbox" type="text" placeholder="Search regexp" autocomplete="off" autocapitalize="none" size=40>
-<button id="focus">Focus</button>
-<button id="ignore">Ignore</button>
-<button id="hide">Hide</button>
-<button id="show">Show</button>
+
 <button id="reset">Reset</button>
-</div>
-<div id="rightbuttons">
-</div>
-</div>
+
+<span id="home">{{.Title}}</span>
+
+<input id="searchbox" type="text" placeholder="Search regexp" autocomplete="off" autocapitalize="none" size=40>
+
+<div id="page">
+
+<div id="errors">{{range .Errors}}<div>{{.}}</div>{{end}}</div>
 
 <div id="graph">
+
+<div id="actionbox">
```

Comment:
I find the UI where the action menu suddenly shows on the very right after you start typing into the search box a bit weird. It would be better if the menu would show closer to the search box. Or (even better IMO) if there would be some sort of a drop-down-looking button (like the Details one) with the actions (maybe disabled until there is something typed into the box).

This is not a deal breaker for the PR, just mentioning something that we could probably do better.